editoast/src/views/infra/auto_fixes.rs

```diff
@@ -122,6 +124,9 @@ pub enum AutoFixesEditoastError {
     )]
     #[editoast_error(status = 500)]
     MaximumIterationReached(),
+    #[error("Failed during fixes application trial")]
+    #[editoast_error(status = 500)]
+    FixTrialFailure(InternalError),
```

Comment:
Making this a named field allows adding context to the returned error. You can expand the `EditoastError` macro for more details (or check the macro doc).

```suggestion
    FixTrialFailure{fail: InternalError},
```

Reply:
Done in https://github.com/osrd-project/osrd/pull/5844/commits/259630ef64bf90c7547a88771ad6174f446c6b03
Note that I used `source to match [thiserror](https://docs.rs/thiserror/latest/thiserror/#details)